### PR TITLE
I updated the Token create method in the serializer

### PR DESCRIPTION
**Pull request by segunemma2003**
Allowing user to know if email is verified or not during token creation

### CODE REVIEW

File: djoser/constants.py [hunk 1]
```diff
@@ -12,3 +12,4 @@ class Messages(object):
     INVALID_PASSWORD_ERROR = _("Invalid password.")
     EMAIL_NOT_FOUND = _("User with given email does not exist.")
     CANNOT_CREATE_USER_ERROR = _("Unable to create account.")
+    IN_ACTIVE_ACCOUNT_ERROR = _("Email not verified")
```
Comment (dekoza) [inline djoser/constants.py]: Please change `IN_ACTIVE_ACCOUNT_ERROR` -> `INACTIVE_ACCOUNT_ERROR`here and further down the line

**Comment by dekoza** [comment]
The tests do not pass, please fix bugs or update test scenarios.

**Comment by dekoza** [comment]
Another question arises: is this change necessary? Can you provide a usecase (with a proper test scenario) that shows importance of this change?

**Comment by segunemma2003** [comment]
ok

**Comment by dekoza** [comment]
@segunemma2003 how's the status of this?

**Comment by segunemma2003** [comment]
oh i thought it wasnt necessary

**Pull request closed without merging**